addons/account/models/chart_template.py

```diff
@@ -1215,36 +1215,33 @@ def create(self, vals_list):
     def _convert_tag_syntax_to_orm(self, tags_list):
         """ Repartition lines give the possibility to directly give
         a list of ids to create for tags instead of a list of ORM commands.
-
         This function checks that tags_list uses this syntactic sugar and returns
         an ORM-compliant version of it if it does.
         """
         if tags_list and all(isinstance(elem, int) for elem in tags_list):
             return [(6, False, tags_list)]
-        return tags_list
+        return
```

Comment:
In that case, you are setting `None` for a m2m. I'm not sure the ORM is not crying in that case..